fog/ocall_oram_storage/trusted/src/lib.rs

```diff
@@ -550,11 +550,8 @@ mod helpers {
 
 // This stuff must match edl file
 extern "C" {
-    #[no_mangle]
     fn allocate_oram_storage(count: u64, data_size: u64, meta_size: u64, id: *mut u64);
```

Comment:
Is it actually okay to remove no mangle here? I thought that is required for linkage to work. Is it implied by extern C or something?

Reply:
Yup, I believe it is not needed: https://doc.rust-lang.org/std/keyword.extern.html